src/orquestra/sdk/_base/_qe/_qe_runtime.py

```diff
@@ -816,3 +816,55 @@ def stop_workflow_run(self, run_id: WorkflowRunId) -> None:
             raise exceptions.WorkflowRunCanNotBeTerminated(
                 f"{run_id} cannot be terminated."
             ) from e
+
+    def list_workflow_runs(
+        self,
+        *,
+        limit: Optional[int] = None,
+        prefix: Optional[str] = None,
+        max_age: Optional[timedelta] = None,
+        state: Optional[Union[State, List[State]]] = None,
+    ) -> List[WorkflowRun]:
```

Comment:
**suggestion**: there are errors we expect this method to raise, like `UnauthorizedError`. Could you list them in the docstring? This helps with code and test isolation.

Reply:
done in [f4b0dc9](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/3/commits/f4b0dc9ebbf762c0ef05d129f8b38db383a12c00)